### DISCUSSION TITLE
Show configurable product values in order detail view for admins

### DISCUSSION THREAD

**Discussion by falko-burghausen** [vendurehq/vendure#3036] [Q&A] [answered]
Hi, I followed [this guide](https://docs.vendure.io/guides/how-to/configurable-products/) to implement configurable options for products, which works fine. The only part I'm missing: how can I make sure that the configured option (aka custom field value) is shown in the admin order view and also in the order confirmation email to the customer? Sticking with the example from the guide: how can I display in the admin order view, that a product has been configured with engraving option and the text for engraving?

Thanks for your help!

**Comment by michaelbromley** [comment, marked as answer]
Hi!

In the Admin UI order detail view, you should find that the OrderLine custom fields are available to add to the order line table using the menu at the top right: 
![image](https://github.com/user-attachments/assets/373821b5-6c95-4e90-a228-32bf86ec6112)

In the order confirmation email you'll need to customize the template in /static/email/templates/order-confirmation to display the contents of the custom field(s). See [customizing templates](https://docs.vendure.io/reference/core-plugins/email-plugin/#customizing-templates)

Note that if the custom fields include any relation types, they will not be automatically joined so in that case you'll need to [extend the default order confirmation handler](https://docs.vendure.io/reference/core-plugins/email-plugin/#extending-the-default-email-handler)  and use the `.loadData()` method, inject the `EntityHydrator` and ensure the relation custom field is loaded.

**Reply by falko-burghausen** [reply]
Hi @michaelbromley , thanks for these hints, that helps a lot. 🙏
